webdriver/tests/bidi/session/subscribe/invalid.py

```diff
@@ -8,16 +8,16 @@
 
 
 @pytest.mark.asyncio
-async def test_params_empty(bidi_session, send_blocking_command):
+async def test_params_empty(send_blocking_command):
     with pytest.raises(InvalidArgumentException):
-        response = await send_blocking_command("session.subscribe", {})
```

Comment:
Why do you assign the result to a variable that is not used? Same applies to other tests.